.github/workflows/docker-build.yaml

```diff
@@ -229,10 +261,47 @@ jobs:
             TESTS_ACCEPTANCE_RESULT: ${{ needs.tests-acceptance.result }}
         steps:
             -   name: GIT checkout branch - ${{ github.ref }}
-                uses: actions/checkout@v2
+                uses: actions/checkout@v3
                 with:
                     ref: ${{ github.ref }}
             -   name: Build successful
                 run: |
                     EXIT_CODE=`. .github/get-build-exit-code.sh && getBuildExitCodeBasedOnJobResults ${BUILD_FORK_RESULT} ${STANDARDS_RESULT} ${TESTS_RESULT} ${TESTS_ACCEPTANCE_RESULT} > /dev/null ; echo $?`
                     exit "${EXIT_CODE}"
+    review:
+        if: github.event.pull_request.head.repo.full_name == 'shopsys/shopsys'
+        name: Review
+        needs: build-docker-images
+        runs-on: [self-hosted, linux, review-stage]
+        steps:
+            -   name: Extract branch name to ENV variable BRANCH_NAME
+                run: echo "BRANCH_NAME=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}" >> $GITHUB_ENV
+            -   name: Create branch directory
+                run: |
+                    rm -rf ${BRANCH_NAME}
+                    mkdir ${BRANCH_NAME}
+            -   name: Set ENV variables required for review
+                run: |
+                    HOST=${BRANCH_NAME}.${{ vars.REVIEW_STAGE_URL }}
+                    echo "HOSTS=${HOST}, en.${HOST}" >> $GITHUB_ENV
```

Comment:
scond domain should be `en`? in traefik hosts I see `cz`.

Reply:
Second should be cz - fixed